supernode/daemon/mgr/task/manager_util_test.go

```diff
@@ -36,7 +37,7 @@ func init() {
 type TaskUtilTestSuite struct {
 	mockCtl          *gomock.Controller
 	mockCDNMgr       *mock.MockCDNMgr
-	mockDfgetTaskMgr *mock.MockDfgetTaskMgr
+	dfgetTaskMgr     *dfgettask.Manager
```

Comment:
Use the mock mgr instead. And you can update the code about mock by exec the shell `hack/generate-super-mock-mgr`.